web/static/js/actions/questionnaire.js

```diff
@@ -31,6 +32,7 @@ export const SET_DEFAULT_LANGUAGE = 'QUESTIONNAIRE_SET_DEFAULT_LANGUAGE'
 export const SET_ACTIVE_LANGUAGE = 'QUESTIONNAIRE_SET_ACTIVE_LANGUAGE'
 export const SET_SMS_QUESTIONNAIRE_MSG = 'QUESTIONNAIRE_SMS_SET_QUESTIONNAIRE_MSG'
 export const SET_IVR_QUESTIONNAIRE_MSG = 'QUESTIONNAIRE_IVR_SET_QUESTIONNAIRE_MSG'
+export const SET_MOBILE_WEB_QUESTIONNAIRE_MSG = 'QUESTIONNAIRE_MOBILE_WEB_QUESTIONNAIRE_MSG'
```

Comment:
Use `QUESTIONNAIRE_MOBILE_WEB_SET_QUESTIONNAIRE_MSG` for consistency with the previous 2